code/src/WijmoProvider/Features/ToolTip.ts

```diff
@@ -56,25 +56,34 @@ namespace WijmoProvider.Feature {
                     );
                 }
             } else if (ht.cellType === wijmo.grid.CellType.ColumnHeader) {
-                if (
+                const rendered = this._grid.getColumn(ht.getColumn().binding)
+                    .config;
+                this._grid.provider.columns[ht.col] || {};
+
+                if (_currTarget && rendered.headerTooltip) {
+                    if (document.getElementById(rendered.headerTooltip)) {
```

Comment:
Can you please add a comment here?

Reply:
setTooltip wijmo method, allows us to render the content of another element (by Id). If an user wants to have a custom content we can "offer" this workaround

Reply:
I think Bruno meant to add a comment on the code 😄

Reply:
I did understand ahah